sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md

```diff
@@ -12,16 +12,94 @@
 
 ## 1.0.0-beta.1 (2023-11-16)
 
-Version 1.0.0-beta.1 is a preview of our efforts in creating an Azure AI Document Intelligence client library that is developer-friendly
-and idiomatic to the Java ecosystem. The principles that guide
-our efforts can be found in the [Azure SDK Design Guidelines for Java](https://azure.github.io/azure-sdk/java_introduction.html).
+_**Note: [Form Recognizer has been rebranded to Document Intelligence](https://mixedrealitywiki.com/display/VTI/Document+Intelligence+2023-10-31-preview#DocumentIntelligence20231031preview-Rebranding)**_
```

Comment:
this link won't be accessible to customers as far as I understand, I think we should omit this